pkg/schemapatcher/gen_integration_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("CRD Patching From Parsing to Editing", func() {
 		var crdSchemaGen genall.Generator = &Generator{
 			ManifestsPath: "./invalid",
 		}
-		rt, err := genall.Generators{&crdSchemaGen}.ForRoots("./...")
+		rt, err := genall.Generators{&crdSchemaGen}.ForRoots("./apis/kubebuilder/...", "./apis/legacy/...")
```

Comment:
Why did you need to change it for the specific pkgs?

Reply:
This is because I made the (probably wrong!) decision to reuse the contents of `pkg/schemapatcher/testdata` for both the existing tests and the new testing of multiple-packages-per-group. I'll separate the test data for those situations.